recipe/meta.yaml

```diff
@@ -1,33 +1,41 @@
-{% set version = "8.0.382" %}
-{% set zulu_build = "8.72.0.17-ca" %}
+{% set version = "8.0.412" %}
+{% set zulu_build = "8.78.0.19-ca" %}
 
 {% set zulu_url = "https://cdn.azul.com/zulu/bin" %}
-{% set zulu_url = "https://cdn.azul.com/zulu-embedded/bin" %}  # [aarch64]
 {% set zulu_base = "zulu" ~ zulu_build ~ "-jdk" ~ version %}
 
 package:
   name: openjdk
   version: {{ version }}
 
-source:
+source: 
   - url: {{ zulu_url }}/{{ zulu_base }}-linux_x64.tar.gz                      # [linux64]
-    sha256: 7b15a56f2ae005beda0f875a269b3daad14ffa12969306cc745478385b25ba96  # [linux64]
+    sha256: 9c0ac5ebffa61520fee78ead52add0f4edd3b1b54b01b6a17429b719515caf90  # [linux64]
   - url: {{ zulu_url }}/{{ zulu_base }}-linux_aarch64.tar.gz                  # [aarch64]
-    sha256: bde61dd92b47baf9c9a15461c365765ae825b5e6e37602014c457b5be6f7b428  # [aarch64]
-  - url: {{ zulu_url }}/{{ zulu_base }}-macosx_x64.zip                        # [osx and x86_64]
-    sha256: 25bfd1f71562695eda921231fbb2b957d84969afcb814efb299fda85f9d3af51  # [osx and x86_64]
+    sha256: 82c46c65d57e187ef68fdd125ef760eaeb52ebfe1be1a6a251cf5b43cbebc78a  # [aarch64]
+
+  # skipping due to : https://github.com/conda-forge/openjdk-feedstock/pull/162#issuecomment-2091518368
+  # - url: {{ zulu_url }}/{{ zulu_base }}-macosx_x64.tar.gz                     # [osx and x86_64]
+  #   sha256: 2bfa0506196962bddb21a604eaa2b0b39eaf3383d0bdad08bdbe7f42f25d8928  # [osx and x86_64]
+  
```

Comment:
Please undo the changes here (aside from required sha updates, obviously). The skip for osx below is more than enough

Reply:
done! thanks!